docs/source/platform_technical_documentation/ade_custom_extensions/maap_libs.ipynb

```diff
@@ -0,0 +1,58 @@
+{
```

Comment:
<p>Recommend capitalizing instances of "jupyter"</p><p> </p><p>Note:</p><p>When following these instructions, I receive: <code>maap = MAAP(maap_host='api.ops.maap-project.org')</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/146/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>